src/UI/ListView.elm

```diff
@@ -199,6 +200,14 @@ type alias ToggleableCover =
     ToggleableList.Cover
 
 
+type alias Dropdown msg =
+    { toggleMsg : msg, isEnabled : Bool, body : DropdownBody msg }
```

Comment:
What do you think of making it opaque and with a construction function, so in the future, we can use "with-pattern" with it?

Reply:
It is opaque, this type is not exposed at all.